script/run-container.sh

```diff
@@ -125,6 +125,11 @@ container_tnf_kubeconfig_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_
 CONTAINER_TNF_DOCKERCFG=$(join_paths "${container_tnf_dockercfg_paths[@]}")
 container_tnf_dockercfg_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_dockercfg_volume_bindings[@]}")
 
+# Safeguard against an empty variable
+if [[ -n $CONTAINER_TNF_DOCKERCFG ]]; then
+	CONTAINER_TNF_DOCKERCFG="$HOME/.docker/config"
```

Comment:
This makes `$HOME/.docker/config` a mandatory file to exist in the system where our suite runs. What if that file doesn't exist? Wasn't it optional?

Reply:
Yeah the flag is optional but the `-e` variable in the container run command is still empty.  I thought about making this value `CONTAINER_TNF_DOCKERCFG="NA"` or something.  It doesn't have to be the $HOME/.docker/config.

Reply:
Changed to: 
```
if [ -z $CONTAINER_TNF_DOCKERCFG ]; then
	CONTAINER_TNF_DOCKERCFG="NA"
fi
```